src/Pages/Home/Components/Carousel/Carousel.jsx

```diff
@@ -1,16 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Link } from 'react-router-dom';
+import { carouselImgs } from '../../../../Assets/images';
 import styles from './Carousel.module.css';
-import { Carousel1 } from '../../../../Assets/images';
 
 const Carousel = () => {
+	const [currentImg, setCurrentImg] = useState(0);
```

Comment:
For more clarity and readability you can name `currentImage` as `currentImgIndex`.

Reply:
I was going to change that but forgot. Thank you, Ankur for noticing.